man/rgblink.5

```diff
@@ -1,53 +1,37 @@
 .\" SPDX-License-Identifier: MIT
 .\"
-.Dd March 28, 2021
+.Dd December 22, 2023
 .Dt RGBLINK 5
 .Os
 .Sh NAME
 .Nm rgblink
 .Nd linker script file format
 .Sh DESCRIPTION
-The linker script is an external file that allows the user to specify the order of sections at link time and in a centralized manner.
+The linker script is a self-contained file that allows specifying attributes for sections at link time, and in a centralized manner.
+There can only be one linker script per invocation of
+.Nm ,
+but it can be split into several files
+.Pq using the Ic INCLUDE No directive .
+.Ss Basic syntax
+A linker script follows a very line-oriented format: each line can contain up to one directive or section name, and up to one comment.
```

Comment:
Matching the phrasing of rgbasm(5), and trying to avoid the technically-accurate but unusual "up to one" expression.
```suggestion
The linker script syntax is line-based. Each line may have a directive or section name, a comment, both, or neither.
```

Reply:
Fine by me; but please ensure that each line contains no more than one sentence.